resources/views/layouts/block.blade.php

```diff
@@ -1,8 +1,18 @@
 @php
 $print = app('printservice')->isPrintMode();
+$pClass = 'App\Http\Controllers\GenericPagesController';
+if (env('APP_ENV') != 'testing') {
```

Comment:
Consider using `App:environment()` here instead:
https://laravel.com/docs/10.x/configuration#determining-the-current-environment